### PR TITLE
hotfix/1.3.5

### PR DESCRIPTION
**Pull request by builtbywill**
correctly parse `\exponentialE`, not as a function, but as a symbol

### CODE REVIEW

File: tests/exp_test.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def test_exp_func_no_delim():
 
 
 def test_exp_command():
-    assert_equal("\\exponentialE(3)", exp(3))
```
Comment (lily514) [inline tests/exp_test.py:22]: @builtbywill so would \\exponentialE(3) ever be allowed or is that the bug that we should not have allowed it?

Comment (builtbywill) [reply tests/exp_test.py:22]: @lily514 I think technically it would still work but would be treated like e*3

Comment (builtbywill) [reply tests/exp_test.py:22]: But yeah, that is the issue, that we were treating it as a func name instead of the symbol

Comment (builtbywill) [reply tests/exp_test.py:22]: scratch that about it working 😂  no it will not be valid now

Comment (builtbywill) [reply tests/exp_test.py:22]: double scratch that, i was right the first time (apparently I can't understand how unit tests work right now)

Comment (lily514) [reply tests/exp_test.py:22]: I was just thinking it might be valuable to have some more unit tests here modeling what is/is not allowed.

**Comment by jkdufair** [comment]
@builtbywill Would it make sense (and would it even be possible) to treat it as a function if used as a function and as a label or whatever it would otherwise be if not used as a function? I know there's nothing in the repo we forked from about it, so I wonder what would make most sense in terms of how the students would use it. Are there examples in the DB currently of people using it not as a function?

File: tests/exp_test.py (hunk 1 above)
Comment (builtbywill) [reply tests/exp_test.py:22]: @lily514 sure! I can add back the original one with what it actually parses as

**Comment by builtbywill** [comment]
@jkdufair e(..) is just never a valid function, so we were wrong to treat it like one in the first place

**Comment by jkdufair** [comment]
Ok that makes perfect sense. Thank you.

File: tests/exp_test.py (hunk 1 above)
Comment (builtbywill) [reply tests/exp_test.py:22]: @lily514 test cases added/updated!

**Pull request merged**